client/queries/destinationQueries.js

```diff
@@ -0,0 +1,9 @@
+// destinationQueries.js
```

Comment:
did not use proper base branch. these should be in hooks directory like the others on trpc branch